src/main/java/inventorysetups/InventorySetupsPersistentDataManager.java

```diff
@@ -148,6 +162,35 @@ private <T> List<T> loadData(final String configKey, Type type)
 		}
 	}
 
+	private List<InventorySetup> loadV2Setups() {
+		Type setupTypeV2 = new TypeToken<ArrayList<InventorySetupSerializable>>()
+		{
+
+		}.getType();
+		List<InventorySetupSerializable> issList = new ArrayList<>(loadData(CONFIG_KEY_SETUPS_V2, setupTypeV2));
+		List<InventorySetup> isList = new ArrayList<>();
+		for (final InventorySetupSerializable iss : issList)
+		{
+			isList.add(InventorySetupSerializable.convertToInventorySetup(iss));
+		}
+		return isList;
+	}
+
+	private List<InventorySetup> loadV3Setups() {
+		// Keeping this as a distinct function will streamline future migrations.
+		final String wholePrefix = ConfigManager.getWholeKey(CONFIG_GROUP, null, CONFIG_KEY_SETUPS_V3_PREFIX);
+		final List<String> loadedSetupWholeKeys = configManager.getConfigurationKeys(wholePrefix);
+		Set<String> loadedSetupKeys = loadedSetupWholeKeys.stream().map(k -> k.substring(wholePrefix.length()-CONFIG_KEY_SETUPS_V3_PREFIX.length())).collect(Collectors.toSet());

```

Comment:
k => key, also can make this one multiple lines

Reply:
done